src/Evolution/Executables/Cce/CharacteristicExtractBase.hpp

```diff
@@ -88,8 +88,18 @@ struct CharacteristicExtractDefaults {
       Cce::bondi_hypersurface_step_tags,
       tmpl::bind<Cce::integrand_terms_to_compute_for_bondi_variable,
                  tmpl::_1>>>;
-  using cce_integration_independent_tags =
-      tmpl::push_back<Cce::pre_computation_tags, Cce::Tags::DuRDividedByR>;
+  using ccm_matching_tags = tmpl::list<
```

Comment:
I don't understand why these tags are being added in this commit. Can you change the executable in a separate commit or do the changes in this commit break CCE? I feel like the `ccm_matching_tags` could be in a different commit at least. Each commit should do only one logical thing, if more are done you should describe why in the commit message's long/detail section. E.g.:
```
Short description that's under 60 characters

long description here
```

Can you also make commit messages more verby: `Add functions to compute psi0 on the worldtube`

Reply:
Given that the new mutators added to `src/Evolution/Systems/Cce/NewmanPenrose.hpp` are designed to mutate these tags, it would be reasonable to include the tags alongside the mutators. But I'm happy to split them into two commits if you want. 

This won't break CCE. It's just when we run CCM (`uses_partially_flat_cartesian_coordinates=true`, sorry for this long and awkward name, I will explain below), we need to mutate a few more tags. It's hard-coded to be `false` for normal CCE (see `src/Evolution/Executables/Cce/CharacteristicExtract.hpp` line 56).

Reply:
Given that the new mutators added to `src/Evolution/Systems/Cce/NewmanPenrose.hpp` are designed to mutate these tags, it would be reasonable to include the tags alongside the mutators. But I'm happy to split them into two commits if you want. 

This won't break CCE. It's just when we run CCM (`uses_partially_flat_cartesian_coordinates=true`, sorry for this long and awkward name, I will explain below), we need to mutate a few more tags. It's hard-coded to be `false` for normal CCE (see `src/Evolution/Executables/Cce/CharacteristicExtract.hpp` line 56).